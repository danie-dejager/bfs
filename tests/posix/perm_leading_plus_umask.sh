# Test for https://www.austingroupbugs.net/view.php?id=1392
umask 002
bfs_diff perms -perm -+w
