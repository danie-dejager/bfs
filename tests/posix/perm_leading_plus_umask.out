perms/drwxrwxr-x
perms/drwxrwxrwx
perms/f-w--w----
perms/f-w--w--w-
perms/f-wx-wx--x
perms/f-wx-wx-wx
perms/frw-rw-r--
perms/frw-rw-rw-
perms/frwxrwxr-x
perms/frwxrwxrwx
